### PR TITLE
[master][KOGITO-1698]: Add instructions about how to debug the automation tests

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA Ticket: https://issues.redhat.com/browse/KOGITO-1698
Description: Add instructions about how to debug the automation tests using VS Code.
This is running the tests locally using Openshift.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -41,3 +41,52 @@ Check the usage of the tests script:
...
+- Install Delve go module:
+
+```sh
+go get -v github.com/go-delve/delve/cmd/dlv
+```
+
+- Start the Delve process to launch our tests:
+
+```sh
+dlv test --headless --listen=:2345 --log --api-version=2 -- -godog.tags="" -tests.operator-image-name=quay.io/jcarvaja/kogito-cloud-operator -tests.services-image-version=0.9.0-rc2 features/my_feature.feature
```
Comment (MarianMacik) [inline test/README.md:80]: Maybe we can omit the reference to anybody in the README? We can either use there a placeholder or kiegroup, wdyt?

Comment (Sgitario) [reply test/README.md]: Sure, I've already removed the extra parameters here.

File: test/README.md [hunk 2]
```diff
@@ -41,3 +41,52 @@ Check the usage of the tests script:
...
+```
+
+The program will wait until you launch the Debug mode from VS Code (next step).
+
+| It must run in the /test folder
+| Any arguments must be passed after the "--" token, for example: "-- -godog.tags="" -tests.services-image-version=0.9.0-rc2"
+
+- Launch Debug from VS Code
+
+The launch icon is found at the right side of VS Code, open it and then run the Play icon at the top menu of the side bar.
```
Comment (sutaakar) [inline test/README.md:90]: I am not sure what are you referring to here, I launched debug session by selecting Run/Start Debugging from top menu.

Comment (Sgitario) [reply test/README.md:90]: I will rewrite the sentence to make it clearer. Thanks!

Comment (Sgitario) [reply test/README.md]: Updated

**Pull request merged**